test/evergreen.yml

```diff
@@ -2059,6 +2059,29 @@ tasks:
           display_name: " 1 Coverage report main page"
           remote_file: wiredtiger/${build_variant}/${revision}/coverage_report_${build_id}-${execution}/1_coverage_report_main.html
 
+  - name: s3-ext-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: -DENABLE_STRICT=0 -DHAVE_DIAGNOSTIC=1 -DENABLE_S3=1 -DIMPORT_S3_SDK=external -DENABLE_PYTHON=1
```

Comment:
Is `DENABLE_STRICT=0` on purpose?

Reply:
Yes, Unfortunately AWS SDK itself doesn't compile under strict. I have plans to improve on that later with WT-8838.